src/pudl/analysis/ferc1_eia.py

```diff
@@ -1491,8 +1037,12 @@ def add_null_overrides(connects_ferc1_eia):
     ).pipe(
         restrict_train_connections_on_date_range,
         id_col="record_id_ferc1",
-        start_date=min(connects_ferc1_eia.report_date),
-        end_date=max(connects_ferc1_eia.report_date),
+        start_date=min(
```

Comment:
In the case where we aren't running the model on all years of data then we want to restrict the null overrides training data to just the years that we actually have EIA plant parts list data for.

Reply:
ah yes you are so right on here! ty for catching this